server/pulp/server/managers/content/orphan.py

```diff
@@ -237,6 +243,11 @@ def delete_orphans_by_type(content_type_id, content_unit_ids=None):
                 unit_type_id=content_type_id
             ).delete()
             content_units_collection.remove(content_unit['_id'])
+            try:
+                content_model.do_post_delete_actions(content_unit)
+            except AttributeError:
```

Comment:
This worries me only because ``AttributeError`` is a fairly common exception to see, especially from a basic programming error, and it would be very unfortunate if something running inside ``do_post_delete_actions`` raised an unhandled ``AttributeError`` that we never saw.

Maybe it would be better in this case to explicitly check for the method you want before calling it.